mergify_engine/exceptions.py

```diff
@@ -61,6 +61,12 @@ def should_be_ignored(exception: Exception) -> bool:
         ):
             return True
 
+        # NOTE(sileht): an installation return 404, mergify have been uninstalled
+        elif exception.status_code == 404 and str(exception.request.url).endswith(
+            "/access_tokens"
+        ):
```

Comment:
Can't we check that in the place where we do the request? That sounds hackish to do that here. What's the good reason?

Reply:
Yes, this call should be at a single place.